src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -141,6 +141,11 @@ public JToken TransformContent(ErrorBuilder errors, JsonSchema schema, FilePath
             var monikers = _monikerProvider.GetFileLevelMonikers(errors, file);
             var xref = new InternalXrefSpec(uid, href, file, monikers, obj.Parent?.Path);
 
+            if (schema.Properties.TryGetValue("uid", out var uidSchema) && uidSchema.UIDGlobalUnique)
```

Comment:
consider modify the `IsXrefSpec` method to also return `uidSchema` and pass it down here.

Reply:
OK.